### ISSUE TITLE
New SWE-bench submission

### ISSUE THREAD

**Issue by EwoutH** [codestoryai/swe_bench_traces#2]
Hi @theskcd!

As current leaders on the SWE-bench Lite Leaderboard, I was curious if you were planning to submit a new version of Aide, and maybe to multiple leaderboards.

In the [SWE Bench Lite Analysis](https://aide.dev/blog/swe-bench-lite) blog it was noted that:

> **Impossible questions**
> There are a set of impossible questions which cannot be solved by any framework or human. These involve questions where the error needs to be formatted exactly as the test. 

SWE-bench [Verified](https://huggingface.co/datasets/princeton-nlp/SWE-bench_Verified) might help with that. All 500 questions have been human-validated for quality. It might be that Aide could break the 50% threshold as the first model and go into history this way!

**Comment by theskcd** [comment]
definitely! we have been a bit busy right now with working on the editor side of changes and helping our users (the pain of growth)

but we have our eyes on swe-bench-verified and very keen on testing on it
